tables/output_tables.Rmd

```diff
@@ -73,6 +74,66 @@ histology_df %>%
 
 ```
 
+## Table 2: Patients with hypermutant or ultra-hypermutant tumors
+```{r table 2}
+# Read in coding TMB file, merge with histologies data
+tmb_hist <- read_tsv(file.path(data_dir, "pbta-snv-consensus-mutation-tmb-coding.tsv")) %>%
+    # rename TSB to Kids_First_Biospecimen_ID
+  rename(Kids_First_Biospecimen_ID = Tumor_Sample_Barcode) %>%
+  select(Kids_First_Biospecimen_ID, tmb) %>%
+  left_join(histology_df) 
+
+# Gather all PT_IDs (poi = "patients of interest") for those with any hypermutant samples which are not cell lines
+poi <- tmb_hist %>%
+  # filter to the hypermutators
+  dplyr::filter(tmb >= 10) %>%
+  pull(Kids_First_Participant_ID)
+  
+# Gather all BS_IDs (soi = "samples of interest"), whether hypermutant or not, for pts with hypermutant samples
+  soi_df <- tmb_hist %>%
+    filter(Kids_First_Participant_ID %in% poi) %>%
+    # Looks like there is a hypermutant metastatic NBL sample we had not caught previously, but let's remove here since we focus on CNS anyway
+    # keep only WGS samples; remove metastases 
+    filter(experimental_strategy == "WGS" & short_histology != "Metastases") %>%
```

Comment:
Noting this filtering differs from the heatmap. Is this intentional?

Reply:
I noticed this upon making the table in the separate tables RMD file. I created the heatmap using logic from the hypermutator df in [here](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/mutational-signatures/08-explore_hypermutators.Rmd). The initial filtering here was done using an inner join of the coding TMB file and [this exposures file ](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/e1cf58eecdc2e3ddf0f6ad8c145adcccd926b2bb/analyses/mutational-signatures/08-explore_hypermutators.Rmd#L30), which contains cancer groups with >=18 samples, thus the NBL metastasis was not in the hypermutator analysis. [Join code](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/e1cf58eecdc2e3ddf0f6ad8c145adcccd926b2bb/analyses/mutational-signatures/08-explore_hypermutators.Rmd#L50-L67). So, it is a bit different since the coding TMB file is the main file from which I am filtering. I can add a snippet showing the histologies and then explaining removal of the NBL sample to focus on HGG, if that works.

Reply:
Makes sense!